README.md

```diff
@@ -10,91 +10,105 @@ Note: Version 4.0.0 and later require Node 8.
 
 ### Installation
 
-    npm install uwpws
+    npm i uwpws
 
-### Examples
+### Configuration
 
-Import the module and set it's configuration.  You must set a ``cacheMode`` to tell the ``micro-cache`` module to save http requests or not to the filesystem (useful for development).  In production, set that mode to ``wild`` to force all requests to go over the internet.
+Configuration options are passed as an object to the constructor.
 
-#### First, set your config and initialize
+#### `organizationName`
 
-The certInfo object should only contain either a file or an s3 object. If you include both, only the file object will be picked up.
+The organization name is a string that identifies your organization and will appear in the headers of requests to the UW PWS API. This helps when working with UW IT to debug your application.
 
-```JavaScript
-const uwpws = require('uwpws');
-
-let config = {
-  baseUrl:   'https://wseval.s.uw.edu/identity/v2/',
-  cacheExt:  '.json',
-  cacheMode: 'wild',
-  cachePath: '.cache/',
-  certInfo:  {
-    file: {
-      cert: 'PATH TO CERT',
-      key:  'PATH TO KEY'
-    }
-    // s3: {
-    //   certBucket: 'uwfosterit.certs',
-    //   certKey:    'laps_cert.cer',
-    //   keyBucket:  'uwfosterit.certs',
-    //   keyKey:     'laps_cert.key'
-    // }
-  },
-};
+#### `baseUrl`
 
-await uwpws.initialize(config);
-```
+The URL of the PWS server. You can use the test or production server.
+
+#### `auth: { cert: 'cert data', key: 'key data' }`
 
-#### Then, search for a Person or Entity
+The Person Web Service requires that you pass a valid UW x509 client certificate with all requests. The data returned from the request is restricted to what is authorized for your cert.
 
-Search for a person, see ``src/modules/person`` for all the parameters you can use and ``test/unit/person-test.js`` for implementations. Notice, the Person and Entity searches are very similar, both have ``get`` and ``search`` functions.  Simply use ``uwpws.entity`` instead of ``uwpws.person``.
+You can use the included certificate fetcher helpers to get cert and key data from local files, from an AWS S3 bucket, or you can create a custom fetcher.
+
+#### `uwPwsLogLevel`
+
+You can set the log level to `silly`, `trace`, `debug`, `info`, `warn`, `error`, or `fatal`.
 
 ```JavaScript
-// Using Promises
-let options = {
-  id: '24CB6CD8AE3511D68CBC0004AC494FFE'
-};
+import UwPws from 'uwPws';
+
+const config = {
+  organizationName: 'YOUR ORGANIZATION NAME',
+
+  baseUrl: 'https://ws.admin.washington.edu/student/v5/',
+
+  //uwPwsLogLevel: 'debug',
```

Comment:
commented in or removed?

Reply:
It's showing it as a demo that the config option is optional and a subtle hint that it could be useful during development, but not needed in prod or general use.